package.json

```diff
@@ -51,7 +51,8 @@
     "can-globals": "^0.2.2",
     "can-namespace": "^1.0.0",
     "can-param": "^1.0.1",
-    "can-parse-uri": "^1.0.0"
+    "can-parse-uri": "^1.0.0",
+    "can-util": "^3.10.8"
```

Comment:
This would cause a cyclic dependency between can-util and can-ajax, which makes things difficult for package maintainers.

Instead of deepAssign from can-util, instead import can-reflect and use canReflect.assignDeep().